Sources/StreamChatTestTools/ChatClient_Mock.swift

```diff
@@ -11,23 +11,7 @@ public extension _ChatClient {
         var config = ChatClientConfig(apiKey: .init("--== Mock ChatClient ==--"))
         config.isLocalStorageEnabled = false
         
-        return .init(
-            config: config,
-            workerBuilders: [],
-            environment: .init(
-                apiClientBuilder: APIClient_Mock.init,
-                webSocketClientBuilder: {
-                    WebSocketClient_Mock(
-                        connectEndpoint: $0,
-                        sessionConfiguration: $1,
-                        requestEncoder: $2,
-                        eventDecoder: $3,
-                        eventNotificationCenter: $4,
-                        internetConnection: $5
-                    )
-                }
-            )
-        )
+        return .init(config: config, tokenProvider: .development(userId: .anonymous))
```

Comment:
You can't just change a piece of code without understanding why was it here ❗ What about the `environment` field and `workerBuilders`?

Reply:
Yes totally missed this, this was also a quick fix, and fix it later, but forgot about this one, will include everything 👍